examples/localstack-spring-boot-sample/pom.xml

```diff
@@ -17,10 +17,10 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.8</java.version>
-		<aws-java-sdk.version>1.11.852</aws-java-sdk.version>
-		<testcontainers.version>1.17.6</testcontainers.version>
-		<localstack-starter.version>0.0.3-SNAPSHOT</localstack-starter.version>
+		<java.version>17</java.version>
+		<aws.sdkv2.version>2.20.143</aws.sdkv2.version>
```

Comment:
I think it is better to keep the name of this property `<aws-java-sdk.version>` and update the version to the latest so that even if there is a v3 version we don't have to keep changing this property all over the place.

Reply:
Yes, thank you! BTW, I've been doing mostly scala for the last several years so my Java/Maven/Spring is rusty, trying to catch up!